src/main/scala/Builders.scala

```diff
@@ -42,6 +43,31 @@ trait Builder[T] {
   override def toString: String = build.toString
 }
 
+trait BlockContentBuilder {
+  def withNote(x: Note): this.type = withBlockContent(x)
+  def withFootnote(x: Fn): this.type = withBlockContent(x)
+  def withMedia(x: Media): this.type = withBlockContent(x)
+  def withParagraph(x: P): this.type = withBlockContent(x)
+  def withTable(x: Table): this.type = withBlockContent(x)
+  def withBlockQuote(x: Bq): this.type = withBlockContent(x)
+  def withOrderedList(x: Ol): this.type = withBlockContent(x)
+  def withPreformatted(x: Pre): this.type = withBlockContent(x)
+  def withUnorderedList(x: Ul): this.type = withBlockContent(x)
+  def withDefinitionList(x: Dl): this.type = withBlockContent(x)
+  def withHorizontalRule(x: Hr): this.type = withBlockContent(x)
+  def withNitfTable(x: NitfTable): this.type = withBlockContent(x)
+  def withSubordinateHeadline(x: Hl2): this.type = withBlockContent(x)
+
+  def withTextParagraph(x: String, markAsSummary: Boolean = true): this.type = {
```

Comment:
This is a new helper method in this class.  It used to exist as `BodyHeadBuilder#withAbstract(String, Boolean)`.